search-app/src/cmr/search/routes.clj

```diff
@@ -62,6 +62,31 @@
                 :body-copy body
                 :body (java.io.ByteArrayInputStream. (.getBytes body)))))))
 
+
+(defn add-equal-sign-to-params-in-list
```

Comment:
We should add unit tests for this function.

Reply:
Done.